samples/bluetooth/hci_vs_scan_req/prj.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_BT=y
+CONFIG_LOG=y
+CONFIG_BT_BROADCASTER=y
+CONFIG_BT_DEVICE_NAME="VS Scan Notify"
+CONFIG_BT_CTLR_VS_SCAN_REQ_RX=y
+CONFIG_BT_CTLR_ADVANCED_FEATURES=y
+CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL=y
```

Comment:
These should be placed in a separate overlay file, say `overlay-bt_ll_sw_split.conf` and have a `sample.yaml` file that be by CI to test building this sample (take a look at other samples that do something similar).

Reply:
I pushed you change proposal. The sample app builds on my machine. I don't know why it's getting this error in CI:
`FileNotFoundError: [Errno 2] No such file or directory: '_test_plan_partial.json'`
When I try running on my console:
`scripts/twister -c -T tests/bluetooth/init -T samples/bluetooth/hci_vs_scan_req --all --save-tests _test_plan_partial.json`
I see no errors, yet not sure if it's done correctly.